gen3-client/g3cmd/configure.go

```diff
@@ -66,5 +79,7 @@ func init() {
 	configureCmd.MarkFlagRequired("cred")
 	configureCmd.Flags().StringVar(&apiEndpoint, "apiendpoint", "", "Specify the API endpoint of the data commons")
 	configureCmd.MarkFlagRequired("apiendpoint")
+	configureCmd.Flags().StringVar(&useShepherd, "use-shepherd", "", fmt.Sprintf("Enables or disables support for the Shepherd API. If enabled, gen3client will use the Shepherd API if available. (Default: %v)", commonUtils.DefaultUseShepherd))
+	configureCmd.Flags().StringVar(&minShepherdVersion, "min-shepherd-version", "", fmt.Sprintf("Specify the minimum version of Shepherd that the gen3client will use if Shepherd is enabled. (Default: %v)", commonUtils.DefaultMinShepherdVersion))
```

Comment:
put this into docs as well?

Reply:
I put a section in the README, anything else I should add? https://github.com/uc-cdis/cdis-data-client/pull/73/files#diff-04c6e90faac2675aa89e2176d2eec7d8R41-R47

Reply:
oh it is just the README seems don't have anything about the `min-shepherd-version` arg 😅

Reply:
Oh gotchu, yes I'll add that!